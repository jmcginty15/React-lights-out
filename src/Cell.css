.Cell {
  height: 100px;
  width: 100px;
  background-color: rgba(50, 50, 50, 0.842);
  cursor: pointer;
  border-radius: 5px;
}

.Cell-lit {
  background-color: rgba(215, 255, 37, 0.87);
}