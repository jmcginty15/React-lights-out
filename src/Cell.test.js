import React from 'react';
import { render } from '@testing-library/react';
import Cell from './Cell';

// smoke test
it('renders without crashing', function () {
    render(<Cell />);
});

// snapshot test
it('matches lit snapshot', function () {
    const { asFragment } = render(<Cell isLit={true} />);
    expect(asFragment()).toMatchSnapshot();
});

it('matches unlit snapshot', function () {
    const { asFragment } = render(<Cell isLit={false} />);
    expect(asFragment()).toMatchSnapshot();
});
