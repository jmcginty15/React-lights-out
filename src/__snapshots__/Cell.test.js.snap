// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches lit snapshot 1`] = `
<DocumentFragment>
  <td
    class="Cell Cell-lit"
  />
</DocumentFragment>
`;

exports[`matches unlit snapshot 1`] = `
<DocumentFragment>
  <td
    class="Cell "
  />
</DocumentFragment>
`;
